remo/domain/dataset.py

```diff
@@ -271,12 +291,13 @@ def get_annotation_set(self, annotation_set_id: int = None) -> AnnotationSet:
         """
         if not annotation_set_id:
             return self.default_annotation_set()
-
-        annotation_set = self.sdk.get_annotation_set(annotation_set_id)
-        if annotation_set and annotation_set.dataset_id == self.id:
-            return annotation_set
-        else:
-            raise Exception('Annotation set with ID = {} is not part of dataset {}. You can check the list of annotation sets in your dataset using dataset.annotation_sets()'.format(annotation_set_id, self.name))
+        
+        if annotation_set_id != -1:
```

Comment:
when `annotation_set_id is -1`?

Reply:
@vmanilo I think is bad design on my side. Need to find a way to tell the function to create a new annotation set